src/GraphRequest.ts

```diff
@@ -430,7 +423,7 @@ export class GraphRequest {
 
 	/**
 	 * @public
-	 * To add query string for filter OData Query param
+	 * To add query string for filter OData Query param. The request URL accepts only one $filter Odata Query option and it's value is set to the most recently passed filter query string.
```

Comment:
nit: typo `it's` -> `its` in this and similar comments below

Reply:
Fixed the spelling error